src/helper/Reply.cc

```diff
@@ -176,7 +228,11 @@ Helper::Reply::parseResponseKeys()
         if (v != nullptr && urlDecode && (p-v) > 2) // 1-octet %-escaped requires 3 bytes
             rfc1738_unescape(v);
 
-        notes.add(key, v ? v : ""); // value can be empty, but must not be NULL
+        // TODO: Convert the above code to use Tokenizer and SBuf
+        const SBuf parsedKey(key);
+        const SBuf parsedValue(v ? v : ""); // allow empty values
```

Comment:
Isn't SBuf(nullptr) the same as SBuf("")?

If so, why the need for the test?

Reply:
> Isn't SBuf(nullptr) the same as SBuf("")?

Yes, the primary effect is the same AFAICT.

> If so, why the need for the test?

We do not. I removed that no longer necessary test in https://github.com/squid-cache/squid/commit/d1f196d4af711f7b5a9413a805fb8deddac16a01